### PR TITLE
EASY-2127 update README.md and test-resources 

### PR DESCRIPTION
**Pull request by rkruithof**
fixes EASY-2127

#### When applied it will
* update the readme to provide a more comprehensive introduction
* copy two new examples to the noaccess-flow directory
* update all test doi's to conform to Datacite's new standard

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,129 +1,127 @@
...
-of the maven project and then type the following (for `SimpleDeposit`): 
+### Example Bags
 
-    mvn clean install # Only necessary if the code was not previously built.
-    ./run.sh Simple https://act.easy.dans.knaw.nl/sword2/collection/1 myuser mypassword src/main/resources/examples/medium
-    
-This will execute the helperscript `run.sh` which uses a Maven plug-in to run the program using the artifacts in the `target` directory.
-You can select a different example program by replacing `Simple` with `Continued`, `SequenceSimple`, etc. The examples that use 
-continued depositing require an extra argument, the chunk size, before the bag directory, for example:
+Examples of bags which meet the specifications of the SWORD depositing interface can be found in the [resources directory]. These bags are categorized by the flow which they are designed for.
```
Comment (stephanHogenboom) [inline README.md]:
```suggestion
Examples of bags which meet the specifications of the SWORD depositing interface can be found in the [resources directory](tree/master/src/main/resources). These bags are categorized by the flow which they are designed for.
```

File: README.md [hunk 2]
```diff
@@ -1,129 +1,127 @@
...
+The primary difference between the no-access flow and the other flows is that the no-access flow requires a DOI to be present in the metadata.xml.
+The other flows only accept a DOI in the metadata if the bag is intended as an update to the existing dataset.
 
-    ./run.sh Continued https://act.easy.dans.knaw.nl/sword2/collection/1 myuser mypassword 512000 src/main/resources/examples/medium
-    
-For the `SequenceXxx`-examples, the sequence of bags to deposit is the variable number of arguments at the end of the argument list.    
+[resources directory]: https://github.com/DANS-KNAW/easy-sword2-dans-examples/tree/master/src/main/resources
```
Comment (stephanHogenboom) [inline README.md]:
```suggestion
[resources directory](tree/master/src/main/resources)
```

File: README.md [hunk 3]
```diff
@@ -1,129 +1,127 @@
...
+	
+### Deposit Packaging
 
-### Running from the Project Directory
+The easy-sword2 service requires deposits to be sent as zipped bags (see [BagIt]).
+The EASY archive adds some extra requirements.
+These are documented on the [EASY SWORDv2 deposit documentation page] and [DANS BagIt Profile].
+A command line tool called [xmllint] can be used to validate xml files locally.
+  
+[EASY SWORDv2 deposit documentation page]: https://easy.dans.knaw.nl/doc/sword2.html
```
Comment (stephanHogenboom) [inline README.md]:
```suggestion
[EASY SWORDv2 deposit documentation page](https://easy.dans.knaw.nl/doc/sword2.html)
```

File: README.md [hunk 4]
```diff
@@ -1,129 +1,127 @@
...
+[BagIt]: https://datatracker.ietf.org/doc/draft-kunze-bagit
```
Comment (stephanHogenboom) [inline README.md]:
```suggestion
[BagIt](https://datatracker.ietf.org/doc/draft-kunze-bagit)
```

File: README.md [hunk 5]
```diff
@@ -1,129 +1,127 @@
...
+[DANS BagIt Profile]: https://github.com/DANS-KNAW/dans-bagit-profile/blob/master/versions/0.0.0.md
```
Comment (stephanHogenboom) [inline README.md]:
```suggestion
[DANS BagIt Profile](../dans-bagit-profile/blob/master/versions/0.0.0.md)
```

File: src/main/resources/README.md [hunk 6]
```diff
@@ -2,8 +2,9 @@ Example Resources
...
-This distinction is made because No-Access flow expects the depositor to provide their own DOI in the dataset metadata. Every other flow only allows for a DOI with the DANS prefix to be submitted and that is minted by DANS. If no such DOI is provided in the `metadata/dataset.xml` the service will mint and register a new DOI with the DANS prefix for the dataset.
+They are sorted based on the type of flow that the user may be configured for, please refer to your DANS account manager to determine which flow your account has been configured for.
+
+The distinction between flows is made because No-Access flow expects the depositor to provide their own DOI in the dataset metadata. Every other flow only allows for a DOI with the DANS prefix to be submitted and that is minted by DANS. If no such DOI is provided in the `metadata/dataset.xml` the service will mint and register a new DOI with the DANS prefix for the dataset.
 
 [Ingest-flow specs]: https://github.com/DANS-KNAW/easy-specs/blob/master/easy-ingest-flow/easy-ingest-flow.md
```
Comment (stephanHogenboom) [inline src/main/resources/README.md]:
```suggestion
[Ingest-flow specs](../easy-specs/blob/master/easy-ingest-flow/easy-ingest-flow.md)
```

**Review by stephanHogenboom — COMMENTED** [review]
Looks good

File: README.md [hunk 7]
```diff
@@ -1,129 +1,127 @@
...
+Getting started
+-------------------------------------
 
-* Java 8 or higher
-* Maven 3.3.3 or higher
+The following is a step-by-step instruction on how to run these examples using the DANS
+acceptance test server at https://act.easy.dans.knaw.nl/. 
 
-It may also work with lower versions of Java and Maven, but this has not been tested.
+1. From your account manager at DANS request access to the acceptance test server. In order to access the acceptance test server your IP address (range) needs to be whitelisted.
```
Comment (janvanmansum) [inline README.md]: Whitelisting is no longer necessary.

Comment (janvanmansum) [inline README.md]:
```suggestion
1. From your account manager at DANS request access to the acceptance test server. The account manager will provide the information necessary to connect.
```

File: README.md [hunk 8]
```diff
@@ -1,129 +1,127 @@
...
+2. Create an EASY account via https://act.easy.dans.knaw.nl/ui/register.
+3. From your account manager at DANS request the account to be enabled for SWORD deposits.
+4. From your account manager at DANS inquire which flow the account is configured for.
```
Comment (janvanmansum) [inline README.md]:
```suggestion
4. From your account manager at DANS inquire which flow (see next section) the account is configured for.
```

File: README.md [hunk 9]
```diff
@@ -1,129 +1,127 @@
...
+3-5. During this time the Client periodically checks the deposit state through the URL received in step 2.
+If the final state of `ARCHIVED` is reached, the process is concluded successfully. Other outcomes may be `INVALID` (the package did not meet the requirements of the SWORD service) or `REJECTED` (the package did not meet the requirements of the EASY Ingest Flow). In case the server encountered an unknown error `FAILED` will be returned.
 
 
-Building
---------
 
-To build the examples you need:
+Getting started
+-------------------------------------
```
Comment (janvanmansum) [inline README.md]:
```suggestion
---------------
```

File: README.md [hunk 10]
```diff
@@ -1,129 +1,127 @@
...
+4. From your account manager at DANS inquire which flow the account is configured for.
 
-Steps:
 
-        git clone https://github.com/DANS-KNAW/easy-sword2-dans-examples.git
-        cd easy-sword2-dans-examples
-        mvn install
+### Flow types
 
+The SWORD service will automatically select a flow for the data when a deposit is submitted. The flow that is selected depends on the agreement between the client company and DANS. The following is a list of the available flows, with a general description of each:
```
Comment (janvanmansum) [inline README.md]:
```suggestion
The SWORD service will automatically select a flow for the data when a deposit is submitted. The flow that is selected depends on the agreement between the depositor organization and DANS. The flow configured for your account will be one of the following:
```

File: README.md [hunk 11]
```diff
@@ -1,129 +1,127 @@
...
 
-Deposit Packaging
------------------
+* `Default` - this flow is only for internal use.
```
Comment (janvanmansum) [inline README.md]:
```suggestion
```

File: README.md [hunk 12]
```diff
@@ -1,129 +1,127 @@
...
+* `Agreement` - this means a Depositor License Agreement Contract is in place. The datasets will be disseminated by DANS.
```
Comment (janvanmansum) [inline README.md]:
```suggestion
* `Agreement` - The datasets will be disseminated by DANS. DANS will mint DOIs for the datasets. 
```

File: README.md [hunk 13]
```diff
@@ -1,129 +1,127 @@
...
+* `NoAccess` - the files are not to be disseminated by DANS.
```
Comment (janvanmansum) [inline README.md]:
```suggestion
* `NoAccess` - the files are not to be disseminated by DANS. The depositor organization must mint DOIs for the datasets.
```

File: README.md [hunk 14]
```diff
@@ -1,129 +1,127 @@
...
+* `Default` - this flow is only for internal use.
+* `Agreement` - this means a Depositor License Agreement Contract is in place. The datasets will be disseminated by DANS.
+* `NoAccess` - the files are not to be disseminated by DANS. 
 
-The [easy-sword2] service requires deposits to be sent as zipped bags (see [BagIt]). The EASY archive adds some
-extra requirements. These are documented on the [EASY SWORDv2 deposit documentation page].
-  
-[EASY SWORDv2 deposit documentation page]: https://easy.dans.knaw.nl/doc/sword2.html
-[BagIt]: https://datatracker.ietf.org/doc/draft-kunze-bagit
+The primary difference between the no-access flow and the other flows is that the no-access flow requires a DOI to be present in the metadata.xml.
```
Comment (janvanmansum) [inline README.md]:
```suggestion
``

**Review by janvanmansum — CHANGES_REQUESTED** [review]
Look already much better! Some things can be improved still, as discussed off-line:

* Extend "Getting started" to include a *concrete* example of running the SimpleDeposit program (with the example deposit to send depending on the flow type of course).
* Move the more general info about the various types of deposits you can send to a sections after "Getting started".
* Check if you can improve the part about output analysis by pretty printing the XML (Search for a way to do that in Java, see for example https://stackoverflow.com/questions/139076/how-to-pretty-print-xml-from-java for a starting point.
* Side issue (as discussed): check what happens when you send an *update* deposit for an Agreement-account and you provide the previously minted DANS-DOI in the dataset.xml.

File: README.md [hunk 15]
```diff
@@ -1,129 +1,127 @@
...
-### Running from the Project Directory
+The easy-sword2 service requires deposits to be sent as zipped bags (see [BagIt]).
+The EASY archive adds some extra requirements.
+These are documented on the [EASY SWORDv2 deposit documentation page] and [DANS BagIt Profile].
+A command line tool called [xmllint] can be used to validate xml files locally.
+  
+[EASY SWORDv2 deposit documentation page]: https://easy.dans.knaw.nl/doc/sword2.html
+[BagIt]: https://datatracker.ietf.org/doc/draft-kunze-bagit
+[DANS BagIt Profile]: https://github.com/DANS-KNAW/dans-bagit-profile/blob/master/versions/0.0.0.md
+[xmllint] : http://xmlsoft.org/xmllint.html
```
Comment (janvanmansum) [inline README.md]:
```suggestion
[xmllint]: http://xmlsoft.org/xmllint.html
```

Always check if your Markdown renders correctly!

**Comment by janvanmansum** [comment]
It is not completely as I would like it, but a major improvement over what we had, so I'll merge it anyway.

**Pull request merged**